bt_m:
  ros__parameters:
    # command : 1: front_arm 2: back_arm 3: arm_move 5: arm spin 180 7: take board
    # material : 0: none 1: whole 2: 1 layer 3: 2 column 2 board 4: 2 layers
    # layer : 4: 2+1 layers

    # command, front_material, back_material
    front_collect: [1, 1, 0]
    back_collect: [2, 0, 1]

    # command, front_material, back_material, finished layer
    construct_1: [1, 3, 0, 0]
    
    construct_2: [
      2, 0, 1, 1,
      7, 0, 0, 1,
      3, 0, 0, 1,
      1, 0, 0, 1,
      3, 2, 0, 1,
      7, 2, 0, 1,
      3, 2, 0, 1,
      1, 2, 0, 1]
    
    construct_3: [
      1, 3, 1, 0,
      2, 0, 1, 1,
      7, 0, 0, 1,
      3, 0, 0, 1,
      1, 0, 0, 1,
      3, 2, 0, 1,
      7, 2, 0, 1,
      3, 2, 0, 1,
      1, 2, 0, 1,
      3, 0, 0, 4,
      1, 0, 0, 4,
      3, 4, 0, 1,
      3, 4, 0, 1,
      1, 4, 0, 1]